src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java

```diff
@@ -108,14 +109,18 @@ public ScoringSubsystem(ShooterIO shooterIo, AimerIO aimerIo, HoodIO hoodIo) {
                         ScoringConstants.getAimerMap(), 0.0, ScoringConstants.aimMaxAngleRadians);
 
         timeToPutAimDown = new InterpolateDouble(ScoringConstants.timeToPutAimDownMap(), 0.0, 2.0);
+
+        aimerAvoidElevator =
+                new InterpolateDouble(
+                        ScoringConstants.aimerAvoidElevatorTable(), 0.0, Math.PI / 2.0);
     }
 
     public void setAction(ScoringAction action) {
         this.action = action;
     }
 
     private void idle() {
-        aimerIo.setAimAngleRad(0, true);
+        aimerIo.setAimAngleRad(0.0, true);
```

Comment:
On line 134 (below) we call setAimAngleRad and target 45 degrees. Maybe we should target the current position to avoid any weird behavior during that state transition.

Reply:
After thinking about this more, we don't need newProfile I think. Our initial conditions will always be put into the trapezoidal profile, so newProfile should effectively always be true and we only re-initialize the profile if the goal position changes. So I recommend that we just get rid of newProfile entirely

Reply:
@jkleiber Not having newProfile causes the arm to lag (it only targets the goal when the robot stops moving).

Reply:
What if you ignore the profile and use the goal directly for small angle changes?

Reply:
That's essentially what !newProfile did but I can try the logic without the actual variable.

Reply:
I guess I am thinking something like this in your goal angle setting function (pseudocode):
```
if (Math.abs(goalAngle - prevGoalAngle) < 10 * Constants.degreesToRadians) {
    useProfile = false;
} else { 
    timer.reset();
    timer.start();
    useProfile = true;
}
```
then in the control step you would have the following logic (pseudocode):
```
double controlSetpoint = goalAngle;
if (useProfile) {
    controlSetpoint = profile.calculate(...)
}
// No else case required because we have the default to be the step change to goalAngle.

// Clamp logic to ensure the control setpoint is within the soft stops
controlSetpoint = MathUtil.clamp(0, maxSoftStop, controlSetpoint);

// PID + FF control
double voltage = feedforward.calculate(controlSetpoint) + controller.calculate(currentAngle, controlSetpoint);

// Voltage output, etc.
```

Reply:
Okay, that makes a lot of sense, thanks

Reply:
What if it didn't use previousGoalAngle and instead used the arm's current position (`if (Math.abs(goalAngle - aimAngleRad) < 10 * Constants.degreesToRadians) {`)? That way small changes would be done without a profile

Reply:
That would likely be better, yeah

Reply:
The reason I didn't recommend the current aim angle was due to the concern that we would suddenly stop using the profile as we approach the goal angle (which would be a problem if the goal angle truly remained constant for some reason). After some thought, I think there is a case to be made for checking goalAngle != prevGoalAngle still (i.e. `goalAngle != prevGoalAngle` && the condition you mentioned above)